R/utilities-simulation.R

```diff
@@ -35,7 +38,7 @@
 #' setParameterValues(parameters = parameter3, values = 1)
 #' parameter2$value == parameter3$value # FALSE#'
 #' @export
-loadSimulation <- function(filePath, loadFromCache = FALSE, addToCache = TRUE) {
+loadSimulation <- function(filePath, loadFromCache = FALSE, addToCache = TRUE, resetIds = TRUE) {

```

Comment:
I have added this parmaeter here. Default is true so by default, ids should be resetted. I do now know of a use case where one would absolutely want to keep the ids but I am keeping it just to be compatible